### DISCUSSION TITLE
General Setting for Profiles or Apps to manage link directing

### DISCUSSION THREAD

**Discussion by counttec** [filips123/PWAsForFirefox#249] [Q&A] [answered]
I currently use a PWA for Messages to text on my desktop. I'd like the ability to configure where links are opened, in this case I'd like the PWA to remain on my texting app, and for the link I click on to be opened in a standard Firefox tab.

**Comment by filips123** [comment, marked as answer]
Enabling "Open out-of-scope URLs in a default browser" in the PWA settings should do this. And you should also add "accounts.google.com" to "Domains always allowed to be opened in the PWA browser".

**Reply by counttec** [reply]
I did not realize that Firefox had native PWA settings, or that the settings menu was different for PWAs. This was a tremendous revelation, Thanks!
